.github/workflows/auto-update-contributors-readme.yml

```diff
@@ -0,0 +1,96 @@
+# GitHub Actions documentation:
+# https://docs.github.com/en/actions
+
+name: Auto-update list of contributors
+
+on:
+  workflow_call:
+
+jobs:
+  update_contributors_list:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+
+      - name: Check if the default branch is protected
+        id: default_branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          defaultBranch="$( gh api /repos/${{ github.repository }} \
+            --method GET \
+            --jq '.default_branch'
+          )"
+
+          isProtected="$( \
+            gh api /repos/${{ github.repository }}/branches/$defaultBranch \
+            --jq '.protected'
+          )"
+
+          echo "::set-output name=is_protected::$isProtected"
```

Comment:
This block might not be needed in the future:
https://github.com/akhilmhdh/contributors-readme-action/issues/37

Reply:
All removed!